tests/translator/input/api_request_model_with_validator.yaml

```diff
@@ -0,0 +1,85 @@
+Resources:
+  HtmlFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: s3://sam-demo-bucket/member_portal.zip
+      Handler: index.gethtml
+      Runtime: nodejs12.x
+      Events:
+        GetHtml:
+          Type: Api
+          Properties:
+            RestApiId: HtmlApi
+            Path: /
+            Method: get
+            RequestModel:
+              Model: User
+              Required: true
+              ValidateBody: true
+              ValidateRequest: true
+
+  HtmlFunctionNoValidation:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: s3://sam-demo-bucket/member_portal.zip
+      Handler: index.gethtml
+      Runtime: nodejs12.x
+      Events:
+        GetHtml:
+          Type: Api
+          Properties:
+            RestApiId: HtmlApi
+            Path: /no-validation
+            Method: get
+            RequestModel:
+              Model: User
+              Required: true
+              ValidateBody: false
+              ValidateRequest: false
+
+  HtmlFunctionMixinValidation:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: s3://sam-demo-bucket/member_portal.zip
+      Handler: index.gethtml
+      Runtime: nodejs12.x
+      Events:
+        GetHtml:
+          Type: Api
+          Properties:
+            RestApiId: HtmlApi
+            Path: /mixin
+            Method: get
+            RequestModel:
+              Model: User
+              Required: true
+              ValidateBody: true
+              ValidateRequest: false
+
+  HtmlFunctionNotDefinedValidation:
```

Comment:
Can you add a function or path in one of the existing ones here that adds the following cases:
* only define ValidateBody
* only define ValidateRequest

Reply:
Added those two cases for each scenario, added only a body for true and false and same for request